kotlinq-core/src/main/kotlin/org/kotlinq/api/AbstractGraphVisitor.kt

```diff
@@ -74,7 +74,7 @@ class AbstractGraphVisitor(
     var nodeListener: (Adapter) -> Unit = empty()
 
     var fragmentListener: GraphVisitor.(Fragment) -> Unit =
```

Comment:
Instead add methods `enterFragment(fragment Fragment)` and `leaveFragment(fragment Fragment)` this breaks encapsulation of visitor pattern.